changelog/20562.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+pki: allowed_domains are compared case-sensitive if they use glob patterns
```

Comment:
```suggestion
secrets/pki: allowed_domains are now compared in a case-insensitive if they use glob patterns
```

Also note that this filename should match the PR number not the issue number. So it should be named `changelog/22126.txt`